.eslintrc.json

```diff
@@ -4,5 +4,19 @@
         "react",
         "jsx-a11y",
         "import"
-    ]
+    ],
+    "settings": {
+        "import/parser": "babel-eslint",
+        "import/resolver": {
+            "webpack": {
+                "config": "webpack.config.js"
+            }
+        }
+    },
+    "rules": {
+        //These rules are in place because eslint import resolver webpack refuses to do its job
```

Comment:
sometimes you must admit defeat, I have indeed been bested by webpack and I am unashamed.